src/System.Diagnostics.DiagnosticSource/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,2 @@
+MembersMustExist : Member 'System.Diagnostics.DiagnosticListener.DefaultListener.get()' does not exist in the implementation but it does exist in the contract.
```

Comment:
Ok, @venkat-raman251 looks like you can remove this baseline and rebase on @vancem's change once merged.